site/src/docs/components/hero/index.mdx

```diff
@@ -0,0 +1,200 @@
+---
+slug: '/components/hero'
+title: 'Hero'
+navTitle: 'Hero '
+---
+
+import { Button, Hero, IconAngleRight, IconShare, IconTrash } from 'hds-react';
+import ExternalLink from '../../../components/ExternalLink';
+import PlaygroundPreview from '../../../components/Playground';
+import TabsLayout from './tabs.mdx';
+import InternalLink from '../../../components/InternalLink';
+
+export default ({ children, pageContext }) => <TabsLayout pageContext={pageContext}>{children}</TabsLayout>;
+
+## Usage
+
+### Example
+
+<PlaygroundPreview>
+  <Hero
+    theme={{
+      '--background-color': 'var(--color-bus)',
+      '--color': '#fff',
+      '--koros-color': 'var(--color-bus)',
+    }}
+    koros={{ type: 'pulse', flipHorizontal: true }}
+  >
+    <Hero.Title>Hero without an image</Hero.Title>
+    <Hero.Text>This is an introduction text. The length is circa two sentences. The suitable total length of the title and introduction is around 230 characters, including spaces. The intro ends with a period.</Hero.Text>
+    <Button
+      variant="secondary"
+      role="link"
+      style={{ '--background-color': '#fff', '--color': 'var(--color-bus)', '--border-color': '#fff' }}
+    >
+      Call to action
+    </Button>
+  </Hero>
+</PlaygroundPreview>
+
+### Principles
+
+- Hero is primarily intended only for landing pages.
+- Hero is highly recommended to be **the first element below the main navigation**.
+
+#### Use the Hero component when you want to:
+
+- Lead the user quickly to the actual content of the service or page.
+- To make a visual impact.
+- Add emotion to the look and feel of the page.
+
+#### Functionalities
+- **Image:** There are both with-image and without-an-image options for Hero. When using a Hero with an image, choose an image that works well on all screen sizes.
+- **Colour:** The background colour is recommended to be the primary colour of the colour palette selected for the site.
+- **Koro shapes:** There are six different koro shapes (basic, beat, calm, pulse, vibration and wave). The koro shape can also cover the Hero image at a 45-degree angle. To learn more about Koro shapes and their customization, refer to <InternalLink href="/components/koros">HDS Koros documentation</InternalLink>.
```

Comment:
Koro shape can be either on bottom of the hero or  at 45-degree angle over Hero image?

Point is that currently it tells it can ALSO be at a 45-degree angle, but it doesn't tell the "original" way of using koro.

Reply:
Thanks! Now the "original" way is mentioned first. I used the term _horizontally_.